docs/root/configuration/http/http_filters/compressor_filter.rst

```diff
@@ -0,0 +1,84 @@
+.. _config_http_filters_compressor:
+
+Compressor
```

Comment:
Please provide an example config that explicitly configures the gzip compressor sub-type. The nested types/configs are very confusing for users.